test/testBrowserLanguage.js

```diff
@@ -0,0 +1,112 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class mockLocales{
```

Comment:
```suggestion
class MockLocales {
```
Classes should be PascalCase